allennlp/modules/__init__.py

```diff
@@ -5,13 +5,11 @@
 :class:`~allennlp.models.model.Model` s.
 """
 
-from allennlp.modules.attention import Attention
```

Comment:
We actually want these to still be here. We typically [import the base `Registrable` class in the module above](https://github.com/allenai/allennlp/blob/master/STYLE.md#module-layout-and-imports).

Reply:
Why? 
I'm happy to do it just wondering why.

Reply:
to make imports cleaner / simpler in your code, so you can do e.g.

```
from allennlp.modules import Attention, FeedForward, ...
```

vs having to do

```
from allennlp.modules.attention import Attention
from allennlp.modules.feedforward import FeedForward
...
```

we do this consistently throughout the codebase, as per the linked style guide

Reply:
Ah cool. 
New to Python, thanks. Will do from now on.